src/kiota/Handlers/Plugin/AddHandler.cs

```diff
@@ -56,6 +69,24 @@ public override async Task<int> InvokeAsync(InvocationContext context)
         Configuration.Generation.Operation = ConsumerOperation.Add;
         if (pluginTypes.Count != 0)
             Configuration.Generation.PluginTypes = pluginTypes.ToHashSet();
+        if (pluginAuthType.HasValue && !string.IsNullOrEmpty(pluginAuthRefId))
+        {
+            var pluginAuthConfig = new PluginAuthConfiguration(pluginAuthRefId);
+            switch (pluginAuthType)
```

Comment:
Maybe this should be a static helper somewhere? Or better yet an implicit conversion operator. This way it's not duplicated between commands.

Reply:
Added helper conversion method in the `PluginAuthConfiguration` class.